pkg/datapath/maps/map.go

```diff
@@ -163,6 +163,11 @@ func (ms *MapSweeper) RemoveDisabledMaps() {
 			"cilium_proxy4"}...)
 	}
 
+	if !option.Config.EnableIPv4FragmentsTracking {
+		maps = append(maps, []string{
+			"cilium_ipv4_frag_datagrams"}...)
```

Comment:
Nit: this can be simplified to:

```suggestion
		maps = append(maps, "cilium_ipv4_frag_datagrams")
```

Reply:
Thanks, I'm pushing the fix